splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/table/TableWrapper.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import ColumnLayout from '@splunk/react-ui/ColumnLayout';
+import TableFilter from '../../components/table/TableFilter';
+import Table from '../../components/table/Table';
+import { TableCaptionComponent } from './TableStyle';
+import Select from '@splunk/react-ui/Select';
+import { getUnifiedConfigs } from '../../util/util';
+
+import PropTypes from 'prop-types';
+import { TableSelectBoxWrapper } from './TableStyle';
+
+function TableWrapper({ isInput, serviceName, rowData }) {
+
+    const [searchText, setSearchText] = useState("");
+    const [searchType, setSearchType] = useState("all");
+
+    /**
+     * 
+     * @param row {Object} row
+     */
+    const changeStatus = (row) => {
+        // let oldData = rowData['sfdc_event_log'];
```

Comment:
@tbalar-splunk  Please remove commented code.

Reply:
I have removed all the commented code. this is for reference as this is going to be implemented in the next PR